src/components/Modals/Tasks/CreateTask.module.css

```
@@ -0,0 +1,46 @@
+.modal {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 30%;
+  transform: translate(-50%, -50%);
+  background-color: var(--white-color);
+  padding: 30px;
+  border: 1px solid var(--lightgray-color);
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-shadow: 0 0 2px 0 var(--lightgray-color);
+}
+
+h3 {
+  margin: 0;
+  padding-bottom: 20px;
+  font-size: 25px;
+  line-height: 25px;
```

use `rem` because it's responsive value

---

Good idea! Reworked:
```suggestion
  font-size: 1.5rem;
  line-height: 1.5rem;
```